README.md

```diff
@@ -1 +1,37 @@
-LOADING...
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly update your README file with the `microverse` template instead of the `react` default README to make the repo more professional. you can find it [here](https://github.com/microverseinc/readme-template)